src/sparseml/transformers/sparsification/trainer.py

```diff
@@ -340,6 +340,9 @@ def compute_loss(
             or not self.manager.enabled
             or not self.manager.distillation_modifiers
         ):
+            inputs = {
```

Comment:
Looks good to me. @natuan I am just curious, when this corner case may occur in practice?